app/src/main/java/com/example/mobile_development/MainActivity.kt

```diff
@@ -0,0 +1,244 @@
+                .padding(10.dp)
+        ) {
+            Text(
+                text = "Log in",
+                modifier = modifier,
+                fontSize = 30.sp,
+                fontWeight = FontWeight.Bold,
+            )
+        }
+
+        Row(
+            modifier = Modifier
+                .size(width = 400.dp, 100.dp)
+                .padding(10.dp)
+        ) {
+            OutlinedTextField(
+                modifier = modifier
+                    .size(width = 400.dp, height = 50.dp)
+                    .fillMaxSize(),
+                value = email,
+                onValueChange = { newValue -> setEmail(newValue) },
+                placeholder = { Text("Email") },
+                isError = !ValidateEmail(),
+                supportingText = {
+                    if (!ValidateEmail()) {
+                        Text(
+                            modifier = Modifier.fillMaxWidth(),
+                            text = "Invalid email",
+                            color = MaterialTheme.colorScheme.error
+                        )
+                    }
+                },
+            )
+        }
+
+        Row(
+            modifier = Modifier
+                .padding(10.dp)
+        ) {
+            OutlinedTextField(
+                modifier = modifier
+                    .size(width = 400.dp, height = 50.dp)
+                    .fillMaxSize(),
+                value = password,
+                onValueChange = { newValue -> setPassword(newValue) },
+                placeholder = { Text("Password") },
+                visualTransformation = if (showPass) VisualTransformation.None else PasswordVisualTransformation(),
+                trailingIcon = {
+                    val (icon, iconColor) = if (showPass) {
+                        Pair(
+                            Icons.Filled.Visibility,
+                            Color(Color.Gray.value)
+                        )
+
+                    } else {
+                        Pair(Icons.Filled.VisibilityOff, Color(Color.Gray.value))
+                    }
+
+                    IconButton(onClick = { setShowPass(!showPass)  }) {
+                        Icon(
+                            icon,
+                            contentDescription = "Visibility",
+                            tint = iconColor
+                        )
+                    }
+                }
+            )
+        }
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight()
+                .padding(vertical = 15.dp),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Button(
+                modifier=modifier.size(width = 300.dp, height = 35.dp),
+                onClick = navigateToHomeScreen) {
+
+                Text(text = "Login")
+            }
+        }
+
+    }
+}
+
+@Composable
+fun HomeScreen(modifier: Modifier = Modifier) {
```

Comment:
This is part of the first homework but it's not an issue to merge this PR and include the screen in the next PR. Just keep in mind that the second screen we are to work on is `ProductDetails`, not `Home`.